src/playlist.tsx

```diff
@@ -34,10 +33,10 @@ export class Playlist extends KalturaPlayer.core.BasePlugin {
     playNextOnError: true
   };
 
-  constructor(name: string, player: KalturaPlayerTypes.Player, config: PlaylistConfig) {
+  constructor(name: string, player: KalturaPlayer, config: PlaylistConfig) {
     super(name, player, config);
-    this._player = player;
-    this._dataManager = new DataManager(this._player, this.logger);
+    this.player = player;
```

Comment:
i think that you don't need this line, because `BasePlugin` already has `this.player`, it's a redundant.